src/content/h1/guides/map-making/blender-level-guide/level-creation-exporting/readme.md

```diff
@@ -0,0 +1,159 @@
+```.alert
+This guide assumes you have already gone through [level creation - beginner][level-creation-beginner] and [level creation - advanced][level-creation-advanced].
+```
+
+# Introduction
+Now that we've created a level, we can start going through the necessary steps for export. This page will describe all the options you can find in the export menu for exporting a level to .JMS
+
+If there are any images that you find difficult to read then try opening the image in a new tab to view it in full resolution.
+
+# Finding the export menu
+We will be starting by finding the export menu for our Blender exporter. Find the File menu in the toolbar on the top left and open it to reveal the contents. In the menu you will see a few options but the only option relevant to you is the menu option labeled *Export*. To export our geometry as a .JMS file you will want to select the option labeled *Halo Jointed Model Skeleton (.jms)*:
+
+![A nice export menu so thoughtfully provided by our benefactors](A.png)
+
+An export menu should appear on your screen after clicking *Halo Jointed Model Skeleton (.jms)*. It should look something like this:
+
+![Plenty of options for you to mess with. Lets dig in!](B.png)
+
+Lets break down each of the options found in the menu
+
+## Game version
+The *Game Version* box only has a single option:
```

Comment:
Not sure what's meant by this line -- it says the box only has a single option but lists options below, and I don't think they are noteworthy to repeat here if they won't be talked about in this article. These paragraphs could instead be shortened to `The *Game Version* option lets us choose our target game, which we will set to *Halo CE*:`